components/textdiff/textdiff.js

```diff
@@ -169,13 +169,23 @@ class TextDiffViewModel {
       // data bind at getPatchCheckBox that is rendered with "html" binding.
       // which is reason why manually updating the html content and refreshing kobinding to have it render...
       if (this.patchLineList) {
-        html = html.replace(/<span class="d2h-code-line-[a-z]+">(\+|-)/g, (match, capture) => {
-          if (this.patchLineList()[index] === undefined) {
-            this.patchLineList()[index] = true;
+        html = html.replace(
+          /<td class="(d2h-(?:del|ins)[^"]*)">([\s\S]*?)<\/td>/g,
+          (match, tdClass, content) => {
+            if (this.patchLineList()[index] === undefined) {
+              this.patchLineList()[index] = true;
+            }
+            content = content.replace(
+              /<span class="d2h-code-line-[a-z]+">(\+|-)<\/span>/g,
+              (match, capture) => {
+                return this.getPatchCheckBox(capture, index, this.patchLineList()[index++]);
+              }
+            );
+            return `<td class="${tdClass}" ${
+              this.editState() === 'patched' ? 'style="cursor:pointer"' : ''
```

Comment:
Could you add a class instead and move this style to CSS?

Reply:
Done!